plugins/services/src/js/pages/task-details/ServiceTaskDetailPage.js

```diff
@@ -16,11 +17,11 @@ class ServiceTaskDetailPage extends React.Component {
     const {id, taskID} = params;
 
     let routePrefix = `/services/overview/${encodeURIComponent(id)}/tasks/${encodeURIComponent(taskID)}`;
-    const tabs = [
+    const tabs = Hooks.applyFilter('serviceTaskDetailPageTabs', [
```

Comment:
What I really would like to see here is `Mount`
This is not something you could do right now, but let's imagine:
```javascript
const tabs = [
  <TabLink label="Details" routePath={routePrefix + '/details'} />,
  <TabLink label="Files" routePath={routePrefix + '/files'} />,
  <TabLink label="Logs" routePath={routePrefix + '/logs'} />,
  <Mount type="ServiceTaskDetailPageTabs" />
];
```

I'm down to implement this with backwards compatibility if @orlandohohmeier gives a 💚